### DISCUSSION TITLE
`unknown command "classroom" for "gh"` (in Windows, installed with choco, )

### DISCUSSION THREAD

**Discussion by fuhrmanator** [community/community#125239] [GitHub Education] [answered]
### Hi everyone,

I'm trying to download all the repos in my assignment, and I get the following command line:
<img width="321" alt="image" src="https://github.com/community/community/assets/7606540/3c2c3e02-74a7-4fcc-ad78-a250e57d9a82">

I installed `gh` in Windows using Choco and it looks to be there. But when I execute the line above, I get:

```text
$ gh classroom clone student-repos -a [redacted]
unknown command "classroom" for "gh"

Usage:  gh <command> <subcommand> [flags]

Available commands:
  alias
  api
  attestation
  auth
  browse
  cache
  co
  codespace
  completion
  config
  extension
  gist
  gpg-key
  issue
  label
  org
  pr
  project
  release
  repo
  ruleset
  run
  search
  secret
  ssh-key
  status
  variable
  workflow
```

My conclusion is that `classroom` isn't a command for windows gh... so I look on https://cli.github.com/manual/gh and I also can't find the `classroom` command. What's wrong with my installation?

**Comment by fuhrmanator** [comment, marked as answer]
The "Learn more about Classroom CLI" link in the Student Repositories dialog sends me to a step "[Clone a student’s assignment repository](https://docs.github.com/en/education/manage-coursework-with-github-classroom/teach-with-github-classroom/using-github-classroom-with-github-cli#clone-a-students-assignment-repository)" which skips over the important step of installing the extension to `gh`: [Using the GitHub Classroom extension with GitHub CLI](https://docs.github.com/en/education/manage-coursework-with-github-classroom/teach-with-github-classroom/using-github-classroom-with-github-cli#using-the-github-classroom-extension-with-github-cli-), especially this step:

`gh extension install github/gh-classroom`

That fixed my problem.

**Reply by wojtekdmyszewicz** [reply]
thanks!
